recipe/meta.yaml

```
@@ -122,6 +122,7 @@ outputs:
         - pyqt >=5.15,<5.16
         - pyqtwebengine >=5.15,<5.16
         - qtconsole >=5.6.1,<5.7.0
+        - __linux  # [linux]
```

I think I was wrong, and we'll need all three because they will all be in the noarch directory.
```suggestion
        - __win    # [win]
        - __osx    # [osx]
        - __linux  # [linux]
```

---

We also need to add `string:` to the `build:` directive. I can't reach it for a code suggestion, but this is what it should look like
```
outputs:
  - name: spyder-base
  - name: spyder
    build:
      noarch: python
      string: "win_pyh{{ PKG_HASH }}_{{ PKG_BUILDNUM }}"    # [win]
      string: "osx_pyh{{ PKG_HASH }}_{{ PKG_BUILDNUM }}"    # [osx]
      string: "linux_pyh{{ PKG_HASH }}_{{ PKG_BUILDNUM }}"  # [linux]
```

---

or we might be able to do:
```
outputs:
  - name: spyder-base
  - name: spyder
    build:
      noarch: python
      string: "linux_pyh{{ PKG_HASH }}_{{ PKG_BUILDNUM }}"  # [linux]

---

> I think I was wrong, and we'll need all three because they will all be in the noarch directory.

Actually, let's try with just `__linux` first, and with just the one `string:`. I think this should just produce two spyder conda packages, one for linux and one for win/osx.